docs/user/manual.adoc

```diff
@@ -232,6 +232,26 @@ The `rust-analyzer` binary can be installed via https://brew.sh/[Homebrew].
 $ brew install rust-analyzer
 ----
 
+=== VS Code or VSCodium in Flatpak
+
+We do not recommend using `rust-analyzer` with a Flatpak version of Code.
+Flatpak applications are sandboxed, which has some unfortunate downsides.
+While the sandbox can be disabled for some directories, `/usr/bin` will always be mounted under `/run/host/usr/bin`.
+This prevents applications from running the system's C compiler, or a system-wide installation of Rust.
+
+If you insist on using Flatpak, the following steps might help:
+
+ - both Rust and `rustup` need to be installed using https://rustup.rs. Distro packages _will not_ work.
+ - before installing `rust-analyzer`, you need to launch Code, open a terminal, run `echo $PATH`
```

Comment:
This works if you already have the extension installed too, you just need to restart VSCode/VSCodium so the Flatseal changes take effect